src/components/ProfileContent/ProfileContent.tsx

```diff
@@ -0,0 +1,241 @@
+
+      if (jobRoleInputValue || user.jobRole) {
+        userData.newFields.jobRole = jobRoleInputValue;
+      }
+      if (countryInputValue || user.country) {
+        userData.newFields.country = countryInputValue;
+      }
+      if (birthdayInputValue || user.birthday) {
+        userData.newFields.birthday = birthdayInputValue;
+      }
+
+      if (errMessage) {
+        return;
+      }
+      firstNameRef.current.value = firstNameRef.current.value.trim();
+      lastNameRef.current.value = lastNameRef.current.value.trim();
+      birthdayRef.current.value = birthdayRef.current.value.trim();
+      countryRef.current.value = countryRef.current.value.trim();
+    }
+
+    dispatch(editUser(userData));
+  };
+
+  return (
+    <Container className="profile-content-container">
+      <Row className="first-row">
+        <Card className="card-content card-box">
+          <Card.Body className="card-content__body card-box__body">
+            <Card.Text className="card-content__text">
+              Some advice text.Some advice text.Some advice text.Some advice
+              text.Some advice text.Some advice text.Some advice text.Some
+              advice text.
+            </Card.Text>
+            <Card.Title className="card-content__title">Profile</Card.Title>
+          </Card.Body>
+        </Card>
+        <Col sm>
+          <Row>
+            <Col>First Name</Col>
+            <Col>
+              <InputField
+                className="profile-form"
+                reference={firstNameRef}
+                placeholder={user.firstName || ''}
+                defaultValue={user.firstName || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Country</Col>
+            <Col>
+              {' '}
+              <InputField
+                className={`profile-form ${
+                  !user.country && 'profile-form--empty'
+                }`}
+                reference={countryRef}
+                placeholder={user.country || 'your country'}
+                defaultValue={user.country || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Job Role</Col>
+            <Col>
+              {' '}
+              <InputField
+                className={`profile-form ${
+                  !user.jobRole && 'profile-form--empty'
+                }`}
+                reference={jobRoleRef}
+                placeholder={user.jobRole || 'your job role'}
+                defaultValue={user.jobRole || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+        </Col>
+        <Col sm>
+          <Row>
+            <Col>Last Name</Col>
+            <Col>
+              {' '}
+              <InputField
+                className="profile-form"
```

Comment:
Clear the placeholder when user is deleting the current value from the input, because it looks like deletion is not happening.